dev/test/query.ts

```diff
@@ -1277,7 +1280,8 @@ describe('limitToLast() interface', () => {
 
   it('requires at least one ordering constraints', () => {
     const query = firestore.collection('collectionId');
-    expect(() => query.limitToLast(1).get()).to.throw(
+    const result = query.limitToLast(1).get();
+    return expect(result).to.eventually.be.rejectedWith(
```

Comment:
All the refactor in the original PR was meant to ensure that this is thrown as an exception, rather than as a rejected Promise, but it is probably not worth breaking the client for.